source/layouts/layout.haml

```diff
@@ -43,6 +43,6 @@
         = yield
         %footer#site-footer
           .contact
-            For more questions or more information, please contact #{mail_to("contact@eriedayofcode.com")}
+            If you have questions or need more information, send email to #{mail_to("contact@eriedayofcode.com")}
```

Comment:
If you can, that's typically how I do it, but if you don't have time I can do it as well.